code/controllers/subsystem/heartbeat.dm

```diff
@@ -0,0 +1,20 @@
+SUBSYSTEM_DEF(heartbeat)
+	name = "Heartbeat"
+	flags = SS_KEEP_TIMING
+	wait = 30 SECONDS
+	runlevels = RUNLEVEL_INIT | RUNLEVEL_LOBBY | RUNLEVEL_SETUP | RUNLEVEL_GAME | RUNLEVEL_POSTGAME // ALL THE THINGS
```

Comment:
why not just "ALL" to future proof against new runlevels

Reply:
We don't have that define lmao

Reply:
gottem gg